
2022-10-25 03:44:28.520 - [Debug] [Auth] Attacker 147.182.233.56 trying to authenticate with 'none'
2022-10-25 03:44:28.520 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 03:44:28.667 - [Debug] [Auth] Attacker 147.182.233.56 trying to authenticate with 'password'
2022-10-25 03:44:28.668 - [Debug] [Auto Access] Attacker: 147.182.233.56, Threshold: 1, Attempts: 1
2022-10-25 03:44:28.668 - [Debug] [Auto Access] Compromising the honeypot
2022-10-25 03:44:28.668 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-25 03:44:28.735 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-25 03:44:28.736 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-25 03:44:28.736 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.246
2022-10-25 03:44:28.790 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:3591465) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_4',
  containerIp: '10.0.3.208',
  mitmPort: 1003,
  autoAccessFixed: 1
}
2022-10-25 03:45:32.016 - [Info] MITM Version: 2
2022-10-25 03:45:32.018 - [Info] Auto Access Enabled: true
2022-10-25 03:45:32.018 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 03:45:32.025 - [Info] SSH man-in-the-middle server for 10.0.3.208 listening on 10.0.3.1:1003
2022-10-25 03:45:32.981 - [Debug] [Connection] Attacker connected: 120.236.78.194 | Client Identification: SSH-2.0-Go
2022-10-25 03:45:33.529 - [Debug] [Connection] Attacker connected: 185.55.64.228 | Client Identification: SSH-2.0-Go
2022-10-25 03:45:34.318 - [Debug] [Auth] Attacker 185.55.64.228 trying to authenticate with 'none'
2022-10-25 03:45:34.318 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 03:45:34.563 - [Debug] [Auth] Attacker 185.55.64.228 trying to authenticate with 'publickey'
2022-10-25 03:45:34.565 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 03:45:34.565 - [Debug] [Auth] Attacker: 185.55.64.228 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 03:45:34.652 - [Debug] [Connection] Attacker connected: 223.171.91.144 | Client Identification: SSH-2.0-Go
2022-10-25 03:45:34.670 - [Debug] [Auth] Attacker 120.236.78.194 trying to authenticate with 'none'
2022-10-25 03:45:34.670 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 03:45:34.810 - [Debug] [Connection] Attacker closed the connection
2022-10-25 03:45:34.925 - [Debug] [Connection] Attacker connected: 185.55.64.228 | Client Identification: SSH-2.0-Go
2022-10-25 03:45:35.180 - [Debug] [Connection] Attacker connected: 101.43.224.103 | Client Identification: SSH-2.0-Go
2022-10-25 03:45:35.251 - [Debug] [Auth] Attacker 120.236.78.194 trying to authenticate with 'publickey'
2022-10-25 03:45:35.252 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 03:45:35.253 - [Debug] [Auth] Attacker: 120.236.78.194 has so far made 1 attempts. Remaining: 5 attempts
